src/test/ui/rfc-2565-param-attrs/param-attrs-pass.rs

```diff
@@ -0,0 +1,21 @@
+// compile-pass
+
```

Comment:
Finally, I'd like to see tests accounting for function pointers which also considers everything above (i.e. cfg, ...).